test/basic.html

```diff
@@ -181,14 +181,17 @@
       test('value should notify', function(done) {
         var targetValue = 10;
 
-        slider.addEventListener('value-changed', function(e) {
-          assert.equal(e.detail.value, targetValue);
-          done();
-        });
-        
+        slider.addEventListener('value-changed', onValueChanged);
+
         slider.min = 0;
         slider.max = 100;
         slider.value = targetValue;
+
+        function onValueChanged(e) {
+          assert.equal(e.detail.value, targetValue);
+          done();
+          slider.removeEventListener('value-changed', onValueChanged);
```

Comment:
This fixes the error <code>done() called multiple times</code> when running the tests locally.
This callback was called twice, and the call stack was the same in both calls, even if we set slider.value only once.